### ISSUE TITLE
Recommendation about accessibility concerns with gray-on-gray text

### ISSUE THREAD

**Issue by maneesha** [carpentries/docs.carpentries.org#749]
There was an issue in the [carpentries.org repo](https://github.com/carpentries/carpentries.org/issues/136) about accessibilty concerns with the gray on gray text in the table of contents of the handbook.  Moving the issue from there to here.

> In the Carpentries Handbook, the grey on darker grey text (for example the heading "General Resources" and the subsections under each of the headings once you click on them.

**Comment by zkamvar** [comment]
The grey on grey text is still accessible for the menu items, but it is not accessible for the footer and previous/next buttons.

https://wave.webaim.org/report#/https://docs.carpentries.org/topic_folders/assessment/index.html

**Comment by zkamvar** [comment]
Question: how is the website redesign going to affect this resource?

**Comment by brynnelliott** [comment]
@zkamvar it should be fixed with the redesign, accessibility us in the contract for the website.

**Comment by maneesha** [comment]
@brynnelliott  Are you comfortable if we close this issue, and focus on accessibility issues in the new website and handbook?

**Comment by brynnelliott** [comment]
@maneesha Yes, we can close this.
